Source/Extensions/Blazorise.DataGrid/Models/DataGridSortColumn.cs

```diff
@@ -0,0 +1,34 @@
+﻿namespace Blazorise.DataGrid;
+
+/// <summary>
+/// A column sort definition
+/// </summary>
+public class DataGridSortColumn
+{
+    /// <summary>
+    /// Gets or sets the field name of the column used for sorting (see <see cref="DataGridColumn{TItem}.Field"/>).
+    /// </summary>
+    public string Field { get; init; }
+
+    /// <summary>
+    /// Gets or sets direction of the column used for sorting (see <see cref="DataGridColumn{TItem}.SortDirection"/>).
+    /// </summary>
+    public SortDirection SortDirection { get; init; }
+
+    /// <summary>
+    /// Creates a new instance
+    /// </summary>
+    public DataGridSortColumn()
```

Comment:
Is an empty ctor valid?

Reply:
Removed, thanks. 95e004f2e153bb28fc55d1d2adcf60cf1a77de68